### DISCUSSION TITLE
Move text contacts (NOC, Admin) to Contact objects for Providers (like Site in 3.2.0)

### DISCUSSION THREAD

**Discussion by Zeawiel** [netbox-community/netbox#9235] [Ideas]
Hello,

From 3.2.0, the contact information from Sites are moved to contact objects.

I would like to know you though about doing the same for Providers. 
For now there is 2 text fields : "NOC Contact" and "Admin Contact" and a list of Contact.

![image](https://user-images.githubusercontent.com/6941618/165492742-9c5f6c6f-62aa-42b6-a4ae-eeb28d9fb6b2.png)

I don't know if there is other objects that could have the same "issue".

Best regards

**Comment by jeremystretch** [comment]
This has been proposed in #9046 and is currently awaiting milestone assignment.

**Reply by Zeawiel** [reply]
Oh yes, I search to the discussion part, but not in the issues.
Thank you!
